CHANGELOG.md

```diff
@@ -4,6 +4,16 @@ _**Note:** This is in reverse chronological order, so newer entries are added to
 
 ## Swift 5.8
 
+* The Swift compiler now no longer prohibits collection downcast in a cast pattern. Here is the usage of a collection downcast pattern in a `switch` statement:
+
+```swift
+switch arr {
+   case let ds as [Derived]:
+     ()
+   case is [Derived]:
+     ()
+``` 
```

Comment:
Let’s make this self-contained, e.g. you could wrap the statement in a function taking an array of `Any`, and downcast it to an array of integers or a different frequently used type.